app/models/evss_claim_document.rb

```diff
@@ -24,33 +24,39 @@ class EVSSClaimDocument < Common::Base
 
   # rubocop:disable Layout/LineLength
   DOCUMENT_TYPES = {
-    '1489' => 'Hypertension Rapid Ready For Decision Summary PDF',
```

Comment:
These should be ordered by the L value, not the description string

Reply:
Thanks for the feedback! However, the acceptance criteria specify that the dropdown list should be alphabetized. If this refers to visual ordering, sorting by the L value won't achieve the same result. Could you please confirm?

Reply:
This list doesn’t affect the frontend so the way it is ordered doesn’t matter. The frontend has it’s own copy of this list and needs to be updated accordingly

Reply:
That makes sense, let me push the change.